Discovery/discoveryplugin.py

```diff
@@ -362,11 +381,13 @@ def read_config(self):
 
 
     def show_config_dialog(self):
+        dlg = config_dialog.ConfigDialog(self.config_combo)
```

Comment:
Rather than passing config_combo to the ConfigDialog and updating it there, it would be nicer to repopulate the combo box (clear + add items + set current index) when the dialog has been closed.